### ISSUE TITLE
Check failed builds (887546419/887554869)

### ISSUE THREAD

**Issue by kenorb** [EA31337/EA31337-classes#286]
The following builds fails for some reason:

- [887546419](https://github.com/EA31337/EA31337-classes/runs/887546419?check_suite_focus=tru#step:3:1439)

```
d:\a\_actions\fx31337\mql-compile-action\master\dist\index.js:176
              throw new Error('Compilation failed!');
              ^

Error: Compilation failed!
    at d:\a\_actions\fx31337\mql-compile-action\master\dist\index.js:176:21
    at ChildProcess.exithandler (child_process.js:302:5)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

> Condition.mqh(32,7) : error 282: 'Condition' - identifier already used

- [887554869](https://github.com/EA31337/EA31337-classes/runs/887554869?check_suite_focus=true#step:3:1429)

> Condition.mqh(32,7) : error 282: 'Condition' - identifier already used

- Branch: `dev`

**Comment by kenorb** [comment]
The issue is with platform bug (build version), it seems `5.0.0.2361` can compile MQL4 files, but not MQL5, however after upgrade to `5.0.0.2515` it is opposite, MQL5 can compile, but MQL4 not (no longer supported).

**Comment by kenorb** [comment]
Fixed in 2c18a93a631037162e371f392ffb214df57a5046.
